teapots/common/ndk_helper/CMakeLists.txt

```diff
@@ -1,16 +1,40 @@
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11 -Wall -fno-exceptions -fno-rtti")
+# build native_app_glue as a static lib
+cmake_minimum_required(VERSION 3.4.1)
 
-add_library(ndk-helper STATIC
-            gestureDetector.cpp
-            gl3stub.cpp
-            GLContext.cpp
-            interpolator.cpp
-            JNIHelper.cpp
-            perfMonitor.cpp
-            sensorManager.cpp
-            shader.cpp
-            tapCamera.cpp
-            vecmath.cpp)
+include(AndroidNdkModules)
+android_ndk_import_module_native_app_glue()
```

Comment:
Where does this come from? grep doesn't find itself anywhere in this commit or the master branch.